app/components/application-wrapper.js

```diff
@@ -37,6 +37,7 @@ export default Component.extend({
   classNameBindings: [
     'smartBannerVisible',
     'isFandomAppSmartBannerVisible:with-fandom-app-smart-banner',
+    'isCustomSmartBannerVisible:with-fandom-app-smart-banner',
```

Comment:
how about renaming this class to `with-smart-banner`?

Reply:
I would be careful with renaming classes, who knows what adEng is using them for ;p